pages/common/git-checkout-index.md

```diff
@@ -0,0 +1,20 @@
+# git checkout-index
+
+> Copy files from the index to the working tree.
+> More information: <https://git-scm.com/docs/git-checkout-index>.
+
+- Restore any files deleted since last commit:
+
+`git checkout-index --all`
+
+- Restore any files deleted or changed since last commit:
+
+`git checkout-index --all --force`
+
+- Restore any files changed since last commit (ignoring deleted files):
+
+`git checkout-index --all --force --no-create`
+
+- Create an export of the tree at the last commit in the specified directory (the trailing slash is important):
```

Comment:
Create an export reads a little weird to me. 
Maybe "Create and export a tree file at the last...". ?
```suggestion
- Create an export of the tree at the last commit in the specified directory (the trailing slash is important):
```

Reply:
How about just "Export a tree..."?

"Create and export" is technically not right.

Reply:
How about this?
```suggestion
- Export a copy of the entire tree at the last commit in the specified directory (the trailing slash is important):
```

Reply:
Nice! Looks good (way better than my suggestion).